packages/components/src/ExternalLink/ExternalLink.js

```diff
@@ -0,0 +1,36 @@
+import { contextConnect, useContextSystem } from '@wp-g2/context';
+import { FiExternalLink } from '@wp-g2/icons';
+import React from 'react';
+
+import { Icon } from '../Icon';
+import { VisuallyHidden } from '../VisuallyHidden';
+import * as styles from './ExternalLink.styles';
+
+function ExternalLink(props, forwardedRef) {
+	const { children, href, rel, ...otherProps } = useContextSystem(
+		props,
+		'ExternalLink',
+	);
+
+	return (
+		<a
```

Comment:
I think we can use the [`Link`](https://github.com/ItsJonQ/g2/tree/master/packages/components/src/Link) component here :D.

That should simplify prop destructuring a bit.